bot/exts/filters/filtering.py

```diff
@@ -393,7 +393,9 @@ async def _filter_message(self, msg: Message, delta: Optional[int] = None) -> No
                         autoban = reason and "[autoban]" in reason.lower()
                         if not autoban and filter_name == "filter_invites" and isinstance(result, dict):
                             autoban = any(
-                                "[autoban]" in invite_info.get("reason", "").lower() for invite_info in result.values()
+                                "[autoban]" in invite_info["reason"].lower()
+                                for invite_info in result.values()
+                                if invite_info.get("reason")
```

Comment:
Could be `"[autoban]" in invite_info.get("reason", "").lower()` to avoid the if statement.

Reply:
Wait that's already what was happening. How is this different?

Reply:
Never mind... the value could exist and be None.